exchange/adapter_map.go

```diff
@@ -61,7 +61,7 @@ import (
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/openrtb_ext"
 
-	"prebid-server/adapters/playwire_ortb"
+	"https://github.com/intergi/prebid-server/tree/develop/adapters/playwire_ortb"
```

Comment:
@dbruneau Any chance you know how to handle this import? It is our custom adapter code and will not exist at `github.com/prebid/prebid-server/...`

Reply:
I ran into something like this once, and IIRC `"github.com/prebid/prebid-server/adapters/playwire_ortb"` may actually be the right value.  Though it looks like a URL, I think the `github.com/prebid/prebid-server` part is just the module name defined at the top of `go.mod`, and the rest is the local path to the subdirectory.  That seems to match what [this blog post](https://blog.golang.org/using-go-modules) says at the end of the "Creating a new module" section.